packages/core/content-releases/server/src/services/__tests__/release.test.ts

```diff
@@ -430,6 +436,46 @@ describe('release service', () => {
 
       expect(() => releaseService.delete(1)).rejects.toThrow('Release already published');
     });
+
+    it('remove the scheduling if the release is scheduled', async () => {
+      const strapiMock = {
+        ...baseStrapiMock,
+        entityService: {
+          findOne: jest.fn().mockReturnValue({ id: 1, name: 'test', scheduledAt: new Date() }),
+          delete: jest.fn().mockReturnValue({ id: 1, name: 'test' }),
+        },
+        db: {
+          transaction: jest.fn(),
+        },
+      };
+
+      // @ts-expect-error Ignore missing properties
+      const releaseService = createReleaseService({ strapi: strapiMock });
+
+      await releaseService.delete(1);
+
+      expect(mockSchedulingCancel).toHaveBeenCalledWith(1);
+    });
+
+    it('does not remove the scheduling if the release is not scheduled', async () => {
```

Comment:
same here
Do you mean 
```
it('does not remove the scheduling if the release is not deleted', async () => {
```
?

Reply:
No, in both cases the release is deleted. But a deleted release could be scheduled or not.